cmd/virt-handler/virt-handler.go

```diff
@@ -496,6 +499,33 @@ func (app *virtHandlerApp) shouldChangeRateLimiter() {
 	log.Log.V(2).Infof("setting rate limiter to %v QPS and %v Burst", qps, burst)
 }
 
+// Update virt-handler rate limiter
+func (app *virtHandlerApp) shouldInstallKubevirtSeccompProfile() {
+	config := app.clusterConfig.GetConfig()
+	seccompConfg := config.SeccompConfiguration
+	if seccompConfg == nil {
+		return
+	}
+
+	virtualMachineProfile := seccompConfg.VirtualMachineInstanceProfile
+	if virtualMachineProfile == nil {
+		return
+	}
+
+	if virtualMachineProfile.KubevirtProfile == nil {
+		log.DefaultLogger().Info("Kubevirt Seccomp profile is not enabled")
+		return
+	}
+
+	installPath := filepath.Join("proc/1/root", app.KubeletRoot)
```

Comment:
The path should probably be absolute, no? 
```suggestion
	installPath := filepath.Join("/proc/1/root", app.KubeletRoot)
```

Reply:
It should

Reply:
done